### ISSUE TITLE
CrownFall Stylle Shen gone

### ISSUE THREAD

**Issue by xxAriesxx** [ValveSoftware/Dota2-Gameplay#22888]
### Cosmetic Item Name

CrownFall: The resurrection of Shen - Style Unlock

### Description

I managed to complete the challenge, but when I went to apply the style release, it didn't work. The style continues to be blocked and is no longer in my inventory.

### Example Match ID

8041773918

### Screenshots

![Item CrownFall](https://github.com/user-attachments/assets/6a65d8ec-cc7e-457c-95c4-0e4b260788b2)
![Inventory CrownFall](https://github.com/user-attachments/assets/64ec4377-15d9-4602-a3c6-76fbf7e27a78)
![CrownFall](https://github.com/user-attachments/assets/d377cbab-a79f-4b41-9805-d6d79b5721fd)

**Comment by KurisuExe** [comment]
![image](https://github.com/user-attachments/assets/52b05ffc-ff2c-436c-9e50-651cf614940e)

**Comment by Hallievhrun** [comment]
Hello, Aries. Did you perhaps unlock the alternate style of your Arcana cursor? If so, it would use up the last style unlock tool, of which there were only 2 of them, just like Skywrath Mage's. But if you're talking about Queen Imperia's style unlock, you still have to complete all ACTs together with the Pathfinder Sidequests and claim it at the Comodores Club. 
![The Comodores Club](https://github.com/user-attachments/assets/28e3a3db-da29-4835-bb4b-e496ab7a9639)
![Queen Imperia Style Unlock](https://github.com/user-attachments/assets/359c0975-1b31-4775-bec1-450d7d47fbee)

**Comment by xxAriesxx** [comment]
So in this case, do I still need to complete Kunkka's side quests to be able to enable the queen's style? Do the other 6 missions, because I have already used the two release tools, both on the cursor and on the arcana itself. But now rereading your message, do I need to pay to have this look, even though it was supposed to be enabled at the end of the ACT?

**Comment by Hallievhrun** [comment]
Yeah, you need to purchase all 4 Acts Pathfinders Packs and complete them to be able to claim Queen Imperia's style unlock at Kunkka's club.

Servants of Scree'Auk
_Unlocked upon completion of the Main Quest in Act IV_

Queen Imperia
_Unlocked upon completion of all Main Quests and all Side Quests for all four Crownfall Acts._

**Comment by Hallievhrun** [comment]
Yeah, you need to purchase all 4 Acts Pathfinders Packs and complete them to be able to claim Queen Imperia's style unlock at Kunkka's club.

• Servants of Scree'Auk
Unlocked upon completion of the Main Quest in Act IV.

• Queen Imperia
Unlocked upon completion of all Main Quests and all Side Quests for all four Crownfall Acts.

**Comment by twkonefal** [comment]
What a fucking scam. I already paid for the arcana. Nowhere does it mention requiring additional costs to unlock styles. SCAM.
